public/docs/ts/latest/guide/security.jade

```diff
@@ -1,6 +1,7 @@
 block includes
   include ../_util-fns
 :marked
+  # Security
```

Comment:
I removed this heading deliberately. There's already a heading above it that says "Security" from the site template.